src/main/java/org/pih/petl/job/type/RunMultipleJob.java

```diff
@@ -28,22 +33,21 @@ public RunMultipleJob() {
      */
     @Override
     public void execute(final ExecutionContext context) throws Exception {
-        ApplicationConfig appConfig = context.getApplicationConfig();
-        PetlJobConfig config = context.getJobConfig();
-        List<String> jobs = config.getStringList("jobs");
-        boolean parallelExecution = config.getBoolean("parallelExecution");
-        context.setStatus("Executing " + jobs.size() + " in " + (parallelExecution ? "parallel" : "series"));
-        context.setTotalExpected(jobs.size());
-
-        // TODO: Handle the ability to utilize the parallel execution and run jobs in parallel to each other
-        for (String jobPath : jobs) {
-            context.setStatus("Running job: " + jobPath);
-            PetlJobConfig nestedJobConfig = appConfig.getPetlJobConfig(jobPath);
-            ExecutionContext nestedContext = new ExecutionContext(context.getJobExecution(), nestedJobConfig, appConfig);
-            PetlJob job = PetlJobFactory.instantiate(nestedJobConfig);
-            job.execute(nestedContext);
-            // TODO: Periodically update overall context status from nested status, using thread.  See SqlServerImportJob
-            context.setTotalLoaded(context.getTotalLoaded() + 1);
+        JobConfiguration config = context.getJobConfig().getConfiguration();
+
+        List<JsonNode> jobTemplates = config.getList("jobs");
+        ExecutionConfig executionConfig = config.getObject(ExecutionConfig.class, "execution");
+        if (executionConfig == null) {
+            executionConfig = new ExecutionConfig(1, 0, 5, TimeUnit.MINUTES);
```

Comment:
See above.  This is backwards-incompatible with the previous version of this job type, but I don't think anyone has ever actually used this.  I'm bumping up the PETL version to 3.0.0 anyway to reflect this.